doc/src/site/sphinx/configuration.rst

```diff
@@ -0,0 +1,307 @@
+Sparkta Configuration
+*********************
+
+Overview
+========
+
+This section shows how to configure Sparkta through Typesafe's Config standard.
+
+By default the system will use the config file "reference.conf", but you can overwrite this values having and
+application.conf in your classpath. This is resume of all parameters that you can configure:
```

Comment:
**resume** is a false friend ;) It should be `This is the summary of all parameters that you can configure`